setup.py

```diff
@@ -30,7 +30,7 @@ def has_ext_modules(self):
     packages=find_packages(),
     ext_modules=ext_modules,
     url="https://larq.dev/",
-    install_requires=["packaging>=19"],
+    install_requires=["flatbuffers>=1.12", "packaging>=19"],
```

Comment:
We are already depending implicetly on `flatbuffers` through TensorFlow.

Reply:
If we're depending on it implicitly, why do we have to include it here? Is there something special about `>=1.12`?

Reply:
> If we're depending on it implicitly, why do we have to include it here?

Actually my comment here was wrong and the current stable versions of TensorFlow don't depend on `flatbuffers` yet.

>  Is there something special about `>=1.12`?

This is the version [TensorFlow 2.4](https://github.com/tensorflow/tensorflow/blob/572d7a71e3fbd202148e231e5a18d89ec411a046/tensorflow/tools/pip_package/setup.py#L57) will depend on, so I think it is nice to have the same version constraints.